src/GitVersionTfsTask/Tasks/GitVersion/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "git-version",
```

Comment:
changing it but it does not really matter as this file is just here to indicate the dependency to be bundled with the task.